scripts/examples/06-April-Tags/find_apriltags_max_res.py

```diff
@@ -31,47 +30,15 @@
 
 # Note! Unlike find_qrcodes the find_apriltags method does not need lens correction on the image to work.
 
-# The apriltag code supports up to 6 tag families which can be processed at the same time.
-# Returned tag objects will have their tag family and id within the tag family.
-
-tag_families = 0
-tag_families |= image.TAG16H5  # comment out to disable this family
-tag_families |= image.TAG25H7  # comment out to disable this family
-tag_families |= image.TAG25H9  # comment out to disable this family
-tag_families |= image.TAG36H10  # comment out to disable this family
-tag_families |= image.TAG36H11  # comment out to disable this family (default family)
-tag_families |= image.ARTOOLKIT  # comment out to disable this family
-
-# What's the difference between tag families? Well, for example, the TAG16H5 family is effectively
-# a 4x4 square tag. So, this means it can be seen at a longer distance than a TAG36H11 tag which
-# is a 6x6 square tag. However, the lower H value (H5 versus H11) means that the false positive
-# rate for the 4x4 tag is much, much, much, higher than the 6x6 tag. So, unless you have a
-# reason to use the other tags families just use TAG36H11 which is the default family.
-
-
-def family_name(tag):
-    if tag.family() == image.TAG16H5:
-        return "TAG16H5"
-    if tag.family() == image.TAG25H7:
-        return "TAG25H7"
-    if tag.family() == image.TAG25H9:
-        return "TAG25H9"
-    if tag.family() == image.TAG36H10:
-        return "TAG36H10"
-    if tag.family() == image.TAG36H11:
-        return "TAG36H11"
-    if tag.family() == image.ARTOOLKIT:
-        return "ARTOOLKIT"
+# Please use the TAG36H11 tag family for this script - it's the recommended tag family to use.
 
 
 while True:
     clock.tick()
     img = sensor.snapshot()
-    for tag in img.find_apriltags(
-        families=tag_families
-    ):  # defaults to TAG36H11 without "families".
+    for tag in img.find_apriltags():
         img.draw_rectangle(tag.rect(), color=127)
         img.draw_cross(tag.cx(), tag.cy(), color=127)
-        print_args = (family_name(tag), tag.id(), (180 * tag.rotation()) / math.pi)
+        print_args = ("TAG36H11", tag.id(), (180 * tag.rotation()) / math.pi)
```

Comment:
```suggestion
        print_args = ("TAG36H11", tag.id(), (180 * tag.rotation()) / math.pi)
```
But some boards will have other families enabled, maybe return the name with or `tag.family` ? Also converting all of those to attributes will probably save lots of space.

Reply:
Should this be in another PR? Providing the family name as a function is a missing feature. I could make it as an attribute, but, converting all those over should be done at the same time in a release with other breaking changes I think.

Reply:
Okay but the script can use that attribute now.

Reply:
Okay, will make an attribute with the family name string.

Reply:
Actually you should convert them now while we're at it, this way we can see the total size savings ?

Reply:
Done, added changed in another commit for this. But, I cannot test this while on the road.

Reply:
Looks like ~224 bytes or so. I deleted the subscr function though which probably accounts for those savings. Moving to attribute seems to be a minor savings.

Reply:
Yeah I noticed that too was wondering why its needed. Less code is better anyway.